angular/projects/admin-nrpti/src/app/mines/mines-collection-detail/mines-collection-detail-record-row/mines-collection-detail-record-row.component.html

```diff
@@ -0,0 +1,14 @@
+<td scope="row" data-label="Name" class="col-6">{{ rowData.name || '-' }}</td>
+<td scope="row" data-label="Source System" class="col-2">{{ rowData._sourceSystem || '-' }}</td>
+<td scope="row" data-label="Date" class="col-2">{{ formatDate(rowData.date) || '-' }}</td>
+<td scope="row" data-label="Published State" class="col-2 d-flex">
```

Comment:
optional nit: could change `d-flex` to `d-lg-flex` so the publish text drops down on page collapse similar to the collections list page. But as this table is much simpler than the collections list, its not as important really.

Reply:
Like it. Updated.